src/__mocks__/pages/api/puzzles/instances/submit.ts

```diff
@@ -0,0 +1,9 @@
+import * as faker from 'faker';
+
+export const mockAnswer = (): string => {
+  return faker.random.word();
```

Comment:
this should optionally take a puzzle and if a puzzle is given that is MCQ type, it should randomly return one of the MCQ options. If the puzzle is not MCQ type, it should return the puzzles correct answer 50% of the time.

Problem with current implementation is it will essentially *never* be the right answer

Reply:
You're right, I forgot there's another type of inputType!

Reply:
The tests for this will be in submit.test.ts, not really what you said but I tested 2 kinds of inputTyoe

Reply:
Cool